scripts/adder-collator.sh

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env bash
```

Comment:
Should we consider writing some harness code and verifying the prometheus events? Not a big of complex `bash` scripts.

Reply:
That's not a bad idea, but unless there's library support for submitting transactions to replace polkadot-js, it doesn't really get us much. Bash is an ugly language, but it's hard to beat it when your task is fundamentally to coordinate launching a bunch of stuff on the command line.

That said, if there's library support in Rust so it's possible to automate the registration, then that's a game-changer. I just don't know of it, if it exists.

Reply:
I think for now this is ok. Created a tracking issue https://github.com/paritytech/polkadot/issues/1991

Reply:
We support sending transactions from rust tests, there is no need to use polkadot-js for this. Do you even took a look at the integration test of the collator?

Reply:
Yes, but my intent was to run them standalone for debugging purposes, not as part of the test suite. It's easier to reach for `scripts/adder-collator.sh` than `cargo test -p whatever-module -- --nocapture --no-timeout`, or whatever the proper incantation ends up being.

Reply:
I don't really agree with that, usually it should always be run with the the test suite, not only by some subset of devs that are aware of its existance.
The approach of adding shell scripts imho also doesn't scale well beyond 3.

Reply:
And we also have polkadot-launch by @shawntabrizi